docs/developer-docs/daos/nns/proposal-lifecycle.mdx

```diff
@@ -0,0 +1,89 @@
+---
+keywords: [intermediate, governance, concept, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Proposal lifecycle
+
+<MarkdownChipRow labels={["Intermediate", "Governance", "Concept"]} />
+
+A proposal goes through the following lifecycle. First, a proposal is submitted by a governance participants. Then
+other governance participants form a decision and vote on the proposal. Based on these votes, the governance canister derives
+a proposal decision, i.e., the proposal is either adopted or rejected.
+If the proposal is rejects, usually nothing happens. If the proposal is adopted, the agreed on decision is automatically executed
+by the governance canister.
+Let's look at each stage in the proposal lifecycle in a bit more detail.
+
+## Proposal submission
+
+Any eligible neuron can make and submit a proposal.
+To avoid being inundated by useless proposals, a user submitting a proposal has to pay a fee of 10 ICP when submitting a proposal,
+that they will receive back if the proposal is adopted (but that they will not receive back if the proposal is rejected).
+
+After a proposal is submitted, the proposal is created and stored in the governance canister.
+Moreover, the governance canister computes and stores additional information with each proposal.
+First, the governance canister looks at all neurons and creates an (empty) ballot for each eligible neuron that has a
+dissolve delay of at least 6 months. Thereby, each neuron's ballot stores the neuron's voting power in this moment.
+This means that at the time of the proposal's submission a neuron snapshot is taken and all neurons can vote on the proposal
+with the voting power that they have in this moment.
```

Comment:
```suggestion
Any eligible neuron can make and submit a proposal. To avoid being inundated by useless proposals, a user submitting a proposal has to pay a fee of 10 ICP when submitting a proposal, that they will receive back if the proposal is adopted but not receive back if the proposal is rejected.

After a proposal is submitted, the proposal is created and stored in the governance canister. The governance canister computes and stores additional information with each proposal.

First, the governance canister looks at all neurons and creates an empty ballot for each eligible neuron that has a
dissolve delay of at least 6 months, storing the neuron's voting power in this moment. This means that at the time of the proposal's submission, a neuron snapshot is taken and all neurons can vote on the proposal with the voting power that they have at that moment.
```